.github/actions/compile-examples/compile.sh

```diff
@@ -0,0 +1,18 @@
+set -e
+SKETCH="$1"
+BOARD="$2"
+LIBS="$3"
+
+cp -r . /home/blues/Arduino/libraries/Blues_Wireless_Notecard
```

Comment:
What is this step doing?

Reply:
:+1: It was a hangover from the note-arduino CI

Reply:
left over from copypasta from the note-arduino CI